### ISSUE TITLE
Manual check not working

### ISSUE THREAD

**Issue by nickmedwards** [linkedtales/scrapedin#33]
the code for my profileScraper is below, it opens the Chromium window but closes before I can finish the recaptcha. if you know what's going on and if you could fix it that would be amazing.
```
const profileScraper = await scrapedin({
  email: LINKEDIN_EMAIL, password: LINKEDIN_PASSWORD, isHeadless: false, hasToLog: true
})
  .catch(err => {
    console.error(err);
  });
```
this is what it logs and the error message
```
scrapedin: 2019-10-01T13:56:57.535Z info: [login] logging at: https://www.linkedin.com/login
scrapedin: 2019-10-01T13:57:13.591Z warn: [login] successful login element was not found
scrapedin: 2019-10-01T13:57:13.594Z warn: [login] manual check was required
Error: linkedin: manual check was required, verify if your login is properly working manually or report this issue: https://github.com/leonardiwagner/scrapedin/issues
    at page.waitFor.then.catch (/Users/nickedwards/RupieNetwork/node_modules/scrapedin/src/login.js:62:31)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```
there is also another message
`Unhandled Rejection at: Error: Protocol error (Runtime.callFunctionOn): Target closed.`

**Comment by leonardiwagner** [comment]
@nickmedwards  we just released `v1.0.9` please update your version, now you can log in via cookies, see this tutorial: https://github.com/linkedtales/scrapedin/wiki/Using-Cookies-To-Login

if you still wants to login via `email` and `password`  please check this tip: https://github.com/linkedtales/scrapedin/wiki/Tips increasing the `waitTimeMs` on `profileScraper` function and it'll wait for your manual check time

really sorry for the late response, I was too busy working in this new release

**Comment by leonardiwagner** [comment]
Closing due to lack of feedback.
